site/mkdocs.yml

```diff
@@ -78,7 +78,7 @@ markdown_extensions:
       permalink: 🔗
 
 extra:
-  icebergVersion: '1.6.0'
+  icebergVersion: '1.6.1'
```

Comment:
it;s probably good to update `doap.rdf` and `.github/ISSUE_TEMPLATE/iceberg_bug_report.yml` as well
(see https://github.com/apache/iceberg/pull/10966)